src/styles/theme.tsx

```diff
@@ -7,6 +7,13 @@ export const theme = createTheme({
       lineHeight: '34px',
       fontSize: '32px',
       fontWeight: 700,
+      textAlign: 'center',
+      margin: '10px',
+    },
+    body1: {
+      textAlign: 'center',
+      opacity: '50%',
```

Comment:
Opacity should not be part of the theme, you can add it to the shared component's styles